0.4.0 fail to build

Debian unstable amd64
The build fail with
````
Configuring Doxyfile using configuration

../meson.build:274:2: ERROR: run_target got unknown keyword arguments "input", "install", "install_dir", "output"
dh_auto_configure: error: cd obj-x86_64-linux-gnu && LC_ALL=C.UTF-8 meson .. --wrap-mode=nodownload --buildtype=plain --prefix=/usr --sysconfdir=/etc --localstatedir=/var --libdir=lib/x86_64-linux-gnu returned exit code 1
make: *** [debian/rules:4: build] Error 25
````

--- a/meson.build
+++ b/meson.build
@@ -266,12 +266,12 @@ if doxygen.found()
   doxygen_data.set('abs_top_srcdir', meson.source_root())
 
   doxyfile = configure_file(
-    input: meson.source_root() + '/doc/Doxyfile.in',
+    input: 'doc/Doxyfile.in',
     output: 'Doxyfile',
     configuration: doxygen_data,
     install: false)
   doc_dir = join_paths(get_option('datadir'), 'doc', 'libdsm')
-  run_target('doc',
+  custom_target('doc',
              command: [doxygen, doxyfile],
              input: doxyfile,
              output: 'html',

